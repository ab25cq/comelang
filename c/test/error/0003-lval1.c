/*
PATTERN: c:8
PATTERN: lvalue
*/
int
main()
{
	3++;
}
