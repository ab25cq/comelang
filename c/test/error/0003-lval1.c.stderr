++ and -- expects an lvalue at test/error/0003-lval1.c:8:5
    3++;
    ^
